src/database.rs

```diff
@@ -220,5 +241,10 @@ mod test {
         db.clear().await.unwrap();
 
         assert_eq!(db.load_prop("com.test", "/test", 1).await.unwrap(), None);
+
+        // load all props
+        db.store_prop("com.test", "/test", &ser, 1).await.unwrap();
+        db.store_prop("com.test2", "/test", &ser, 1).await.unwrap();
+        assert_eq!(db.load_all_props().await.unwrap().len(), 2);
```

Comment:
I'd test the correctness of the result too, for example, store a single property, call load_all_props to get it and then check every field for correctness